docs/src/architecture/state.md

```diff
@@ -29,7 +29,7 @@ The latest account states - and data for onchain accounts - are recorded in a [s
   <img src="../diagrams/architecture/state/Account_DB.png" width="80%">
 </p>
 
-As described in [Accounts](http://localhost:3000/architecture/accounts.html#account-storage-modes), there are two types of accounts:
+As described in [Accounts](https://0xpolygonmiden.github.io/miden-base/architecture/accounts.html#account-storage-modes), there are two types of accounts:
```

Comment:
I believe the last part of the path should be `account-storage-mode` (without the "s").